repository/kafka/operator/operator.yaml

```diff
@@ -1,11 +1,13 @@
 name: "kafka"
-version: "0.2.1"
-kudoVersion: 0.5.0
+version: "1.0.0"
+kudoVersion: 0.7.5
```

Comment:
Fine for now but we plan to release this with KUDO `1.0.0` right? (or whatever number they 'GA' with?)

Reply:
right